extern/sector-storage/ffiwrapper/types.go

```diff
@@ -53,6 +56,40 @@ type SectorProvider interface {
 	// * returns storiface.ErrSectorNotFound if a requested existing sector doesn't exist
 	// * returns an error when allocate is set, and existing isn't, and the sector exists
 	AcquireSector(ctx context.Context, id storage.SectorRef, existing storiface.SectorFileType, allocate storiface.SectorFileType, ptype storiface.PathType) (storiface.SectorPaths, func(), error)
+	AcquireSectorPaths(ctx context.Context, id storage.SectorRef, existing storiface.SectorFileType, ptype storiface.PathType) (storiface.SectorPaths, func(), error)
 }
 
 var _ SectorProvider = &basicfs.Provider{}
+
+type PrivateSectorInfo = ffi.PrivateSectorInfo
+
+type SortedPrivateSectorInfo = ffi.SortedPrivateSectorInfo
+
+type FallbackChallenges = ffi.FallbackChallenges
+
+type PartitionProof = ffi.PartitionProof
+
+var GenerateSingleVanillaProof = ffi.GenerateSingleVanillaProof
+
+var MergeWindowPoStPartitionProofs = ffi.MergeWindowPoStPartitionProofs
```

Comment:
ffi types should not be used as part of the interface

Reply:
Ok, resubmitted

Reply:
@siriusmz ci is very sad